### ISSUE TITLE
Working Agreement

### ISSUE THREAD

**Issue by wycliffepeart** [zendexa/project#6]

**Comment by iamshellyan** [comment]
@wycliffepeart i love this....honestly these are thing included in documentations?

**Comment by wycliffepeart** [comment]
> @wycliffepeart i love this....honestly these are thing included in documentations?

Yes, we have to provide documentation about our process and the tools and techniques we are using, so that everyone on the team understands the processes that are required to achieve our objective.

In addition, persons on the team might not understand the agile processes that we are using so it's our responsibility to document and teach them about the processes.

**Comment by wycliffepeart** [comment]
@iamshellyan this
